modelci/experimental/model/model_train.py

```diff
@@ -0,0 +1,164 @@
+#!/usr/bin/env python
+# -*- coding: utf-8 -*-
+"""
+Author: Li Yuanming
+Email: yli056@e.ntu.edu.sg
+Date: 1/31/2021
+
+ML model training parameters.
+"""
+import abc
+from enum import Enum
+from typing import Optional, Union, Tuple, List
+
+from pydantic import BaseModel, PositiveInt, PositiveFloat, root_validator, validator
+
+from modelci.types.vo import Status
+
+
+class DataModuleProperty(BaseModel):
+    dataset: str
+    batch_size: int
+    num_workers: Optional[int] = 1
+    data_dir: Optional[str]
```

Comment:
If data_dir is specified, they data format should be specified as well so we can parse the dataset, let's leave out this topic now and discuss the detail latter

Reply:
This can be done via `dataset` parameter

Reply:
Does `dataset` parameter refer to the name of dataset? or dataset type, like VOC format?

Reply:
In my opinion, it refers to the name of the dataset. E.g. `CIFAR10` can be the `CIFAR10` or `CIFAR10`-liked dataset that can be parsed using the CIFAR10 format.